### ISSUE TITLE
Accessibility Buttons in lack discernible text for screen readers

### ISSUE THREAD

**Issue by OliverAdams56** [frappe/erpnext#54920]
### Information about bug

- Automated testing using Axe DevTools identified 10 instances where buttons on the page do not have discernible text.

- Buttons without a text label or an ARIA description are "silent" to screen reader users. Without discernible text, a user navigating with assistive technology cannot identify the purpose or function of the button, making the interface unusable for them.

- This is a violation of WCAG 2.1 Success Criterion 4.1.2: Name, Role, Value, which requires that every user interface component has a name that can be programmatically determined.

- Priority: Critical.
- Reason: This issue prevents a user from interacting with key navigational elements, representing a complete barrier to access.

<img width="392" height="636" alt="Image" src="https://github.com/user-attachments/assets/ef9b4a83-2e4d-4770-b9e0-cee542f16085" />

### Module

other

### Version

v15.108.1 [Latest](https://github.com/frappe/erpnext/releases/latest)

### Installation method

None

### Relevant log output / Stack trace / Full Error Message.

```shell

```

**Comment by OliverAdams56** [comment]
Closing this as the bug is located in the Frappe Framework. I have opened a new issue and PR in that repository
